mesh/mesh.cpp

```diff
@@ -783,10 +784,11 @@ void Mesh::GetLocalQuadToHexTransformation(
    locpm.SetSize(3, 4);
    for (int j = 0; j < 4; j++)
    {
-      const IntegrationPoint &vert = HexVert->IntPoint(hv[qo[j]]);
-      locpm(0, j) = vert.x;
-      locpm(1, j) = vert.y;
-      locpm(2, j) = vert.z;
+      int k = qo[j];
```

Comment:
If you’ll excuse my naivety here, could you possibly explain this fix and the implications of the bug in your fourth bullet point? It seems like a pretty common code path so without understanding the conventions for orientations/transformations very well I am curious how or when it might affect user codes. Thanks!

Reply:
See my comment here: https://github.com/mfem/mfem/pull/2920/files#r962057756.

Reply:
For more on this, see https://github.com/mfem/mfem/pull/2920#discussion_r962083088 and the follow-up commit 5180b3f92.